cli/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+use std::env;
+
+use clap::{crate_name, crate_version, App, AppSettings, SubCommand};
+
+use crate::deploy::deploy::{info, new, terminate, upgrade};
+
+pub mod deploy;
+
+#[tokio::main]
+pub async fn main() {
```

Comment:
main function in `lib.rs`?

Reply:
fixed. https://users.rust-lang.org/t/main-rs-and-lib-rs-at-same-level/42499/2 for reference since I was unaware